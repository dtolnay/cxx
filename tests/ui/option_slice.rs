#[cxx::bridge]
mod ffi {
    extern "Rust" {
        fn f() -> Option<&'static [u8]>;
    }
}

fn f() -> Option<&'static [u8]> {
    unimplemented!()
}

fn main() {}
