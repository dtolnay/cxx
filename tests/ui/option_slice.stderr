error: unsupported element type of Option
 --> tests/ui/option_slice.rs:4:19
  |
4 |         fn f() -> Option<&'static [u8]>;
  |                   ^^^^^^^^^^^^^^^^^^^^^
